armi/operators/tests/test_operators.py

```diff
@@ -122,7 +181,8 @@ def test_workerException(self):
 
 
 if __name__ == "__main__":
-    args = ["mpiexec", "-n", "2", "python", "-m", "unittest"]
+    """args = ["mpiexec", "-n", "2", "python", "-m", "unittest"]
     args += ["armi.tests.test_operators.OperatorTests"]
-    subprocess.call(args)
-    # unittest.main()
+    subprocess.call(args)"""
```

Comment:
This is a bit confusing as a string literal. Kinda looks like a docstring? Probably better to make it a proper comment.